receptai.api/Controllers/SubfoodditUserController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using receptai.api.Extensions;
+using receptai.api.Interfaces;
+using System.Threading.Tasks;
+
+namespace receptai.api.Controllers;
+
+[Route("api/subfooddit-users")]
+[ApiController]
+public class SubfoodditUserController : ControllerBase
+{
+    private readonly ISubfoodditRepository _subfoodditRepository;
+
+    public SubfoodditUserController(ISubfoodditRepository subfoodditRepository)
+    {
+        _subfoodditRepository = subfoodditRepository;
+    }
+
+    [HttpPost]
+    [Authorize]
+    public async Task<IActionResult> AddUser(int subfoodditId)
+    {
+        var userId = User.GetId();
+        var success = await _subfoodditRepository.AddUserToSubfooddit(subfoodditId, userId);
+        if (!success)
+        {
+            return NotFound("Subfooddit or user not found.");
+        }
+
+        return Ok();
+    }
+
+    [HttpDelete("{userId:int}")]
+    public async Task<IActionResult> RemoveUser(int subfoodditId, int userId)
+    {
+        var success = await _subfoodditRepository.RemoveUserFromSubfooddit(subfoodditId, userId);
+        if (!success)
+        {
+            return NotFound("Subfooddit or user not found.");
+        }
+
+        return NoContent();
+    }
```

Comment:
same thing with removing user - should only be done by user

(sry missed this when creating add user comment)

Reply:
Thanks for noticing! My bad, feel quite distracted 😅